src/Employer/Employer.Web/Orchestrators/Part2/TrainingProviderOrchestrator.cs

```diff
@@ -39,10 +41,20 @@ public TrainingProviderOrchestrator(IEmployerVacancyClient client, ITrainingProv
             return vm;
         }
 
+        public async Task<SelectTrainingProviderViewModel> GetSelectTrainingProviderViewModel(SelectTrainingProviderEditModel m)
+        {
+            var vrm = new VacancyRouteModel { EmployerAccountId = m.EmployerAccountId, VacancyId = m.VacancyId };
```

Comment:
Don't need this.  Just use SelectTrainingProviderEditModel

Reply:
This would fall over as it did earlier. It would be calling the method it was already in resulting in the overflow. The signature of the one I mean to call is different and requires a VacancyRouteModel type so is necessary.

Reply:
I see. Ha! Change the name of one for the functions? Cast it?

Reply:
👍 Will cast it